README.md

```diff
@@ -18,41 +18,73 @@ The script is compatible with any Kerlink Wirnet Station gateway running firmare
+
+                                                                [  OK  ]
 ```
 
 In case gateway was running outdated firmware (e.g. `3.5`), the execution would look like this:
 
 ```
-./provision.sh '192.168.188.101' 'https://thethings.example.com'
+./provision.sh '192.168.4.155' 'https://thethings.example.com'
 CPF is not installed, attempt CPF '1.1.6' installation?[y/n]y
 Gateway firmware version is '3.5', which is lower than '3.6' required by CPF, attempt firmware update to '3.6'?[y/n]y
 Downloading firmware from 'https://github.com/TheThingsNetwork/kerlink-station-firmware/releases/download/wirnet-3.6/fwupgrade_wirmav2_wirnet_v3.6.tar.gz' to '/run/user/1000/tmp.3sb4TlkWRc/dota.tar.gz' locally...
 Pushing '/run/user/1000/tmp.3sb4TlkWRc/dota.tar.gz' to '/mnt/fsuser-1/dota/dota.tar.gz'...
 Firmware successfully pushed, reboot gateway to start update?[y/n]y
-Waiting for 192.168.188.101 to reboot............[down]......................................
-# Host 192.168.188.101 found: line 30
+Waiting for 192.168.4.155 to reboot............[down].....................................
+# Host 192.168.4.155 found: line 30
 /home/rvolosatovs/.ssh/known_hosts updated.
 Original contents retained as /home/rvolosatovs/.ssh/known_hosts.old
+The authenticity of host '192.168.4.155 (192.168.4.155)' can't be established.
+ECDSA key fingerprint is SHA256:yr/giQOwLcT8+ZgDrDKg/ca4XMdOfiqSivrPZBW3d9o.
+Are you sure you want to continue connecting (yes/no)? yes
+Warning: Permanently added '192.168.4.155' (ECDSA) to the list of known hosts.
+root@192.168.4.155's password: 
 Downloading firmware from 'https://github.com/TheThingsNetwork/kerlink-station-firmware/releases/download/wirnet-3.6/custo_knetd-4.12.tar.gz' to '/run/user/1000/tmp.SVnxRHfdfD/dota.tar.gz' locally...
 Pushing '/run/user/1000/tmp.SVnxRHfdfD/dota.tar.gz' to '/mnt/fsuser-1/dota/dota.tar.gz'...
+root@192.168.4.155's password: 
 Firmware successfully pushed, reboot gateway to start update?[y/n]y
-Waiting for 192.168.188.101 to reboot............[down]............
+Waiting for 192.168.4.155 to reboot............[down]............
 Downloading firmware from 'https://github.com/TheThingsNetwork/kerlink-station-firmware/releases/download/cpf-1.1.6/dota_cpf_1.1.6-1.tar.gz' to '/run/user/1000/tmp.Qr60nW4dWc/dota.tar.gz' locally...
 Pushing '/run/user/1000/tmp.Qr60nW4dWc/dota.tar.gz' to '/mnt/fsuser-1/dota/dota.tar.gz'...
 Firmware successfully pushed, reboot gateway to start update?[y/n]y
-Waiting for 192.168.188.101 to reboot............[down]....................................
+Waiting for 192.168.4.155 to reboot...........[down]...................................
+Querying available frequency plan templates...
+Available frequency plan templates are:
+0) AS923-JP_8CH
+1) AU915-AU_8CH
+2) EU868-FR_8CH
+3) IN865-IN_8CH
+4) RU864-RU_8CH
+5) RU864-RU_8CH_Legacy
+6) US915-US_8CH
+Please select a frequency plan template to use for lorad configuration.[0-6]2
```

Comment:
Where do these come from? Kerlink?

I think we have to pull this from https://github.com/TheThingsNetwork/lorawan-frequency-plans, at least the index. Then we can use GCS (https://github.com/TheThingsNetwork/lorawan-stack/blob/master/pkg/gatewayconfigurationserver/gatewayconfigurationserver.go#L70) to get the JSON file. This does require an API key with gateway info rights.

Reply:
This is just what is present in the vanilla CPF DOTA - the script then simply symlinks the template to current config. The user is expected to manually adapt the config at a later stage.
We could provide functionality for more advanced configuration(e.g. enabling and configuring class B) and/or get configuration from GCS.
I am not familiar with GCS, but looking at the source code the obvious thing is that the protocol used there has nothing to do with either `lorad`, nor `lorafwd` config required here. (Referring to https://github.com/TheThingsNetwork/lorawan-stack/blob/57a326a0c4f2bc81b896ede300baff2df199baf6/pkg/pfconfig/semtechudp/semtechudp.go#L34-L40)
What is the scope of GCS? It seems that this component is used to configure gateways using data in the stack. Do users configure this in the console? Does it include things like class B parameters etc.? If it is what I think it is, then we could potentially remove all the string replacements etc. and `provision.sh` could just fetch both `lorad` and `lorafwd` configs from GCS given a (GCS) address and a key.
All of that is quite out of scope of this PR. Can you file an issue?

Reply:
> All of that is quite out of scope of this PR

What is in scope is ensuring that the right frequency plan is used. I don't care about vanilla CPF DOTA frequency plans; we have our own.

What is the format of those? Are those basically `global_conf.json`?

As for the `lorad` and `lorafwd` config; they may very well override what is in `global_conf.json`, assuming that CPF uses that format.